### PR TITLE
First Pull Request!

### PR DESCRIPTION
**Pull request by Zelalem1222**
First I used Html and CSS to modify and style the header. And It also has a gitignore file to ignore unnecessary files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Hello-Microverse
\ No newline at end of file
+Project Name 
+  Hello-Microverse
+
+ Built With
+  
+   Html and css
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly make your readme file more descriptive by using the [microverse readme template](https://github.com/microverseinc/readme-template) this make your project more descriptive and professional 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Zelalem1222,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used




## Required Changes ♻️

- [ ]  Kindly setup [GitHub actions ](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-github-actions) by adding the  .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml), you can also  add [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) and [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) to your root project so you can set up linters locally, also don't forget to add node_modules to .gitignore so you don't add it to git 🤝 

- [ ] Kindly add a  descriptive pull request message as this is part of the requirement, it should be a brief summary describing changes introduced in your pr.

_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```
Comment (MeqdamAlqudah) [inline .github]: - [ ] Please  linters are not working as I noticed and as the last reviewer told you she described the problem " Kindly setup [GitHub actions ](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-github-actions)by adding the .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml), you can also add [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) and [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) to your root project so you can set up linters locally, also don't forget to add node_modules to .gitignore so you don't add it to git 🤝"

there should be 3 checks here for web hint ,silent and Lighthouse

File: .github [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+on: pull_request
+
```
Comment (MeqdamAlqudah) [inline .github]: - [Optional ] Please commit should be in the present tense 

image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Zelalem1222 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .github (hunk 2 above)
Comment (Zelalem1222) [reply .github]: I have already created .github folder and added a copy of [.github/workflows/linters.yml]. Is there anything I have to add?

**Comment by Zelalem1222** [comment]
Hi, @prolajumokeoni
 Can you please help me? I do not know what I am doing wrong.

**Comment by prolajumokeoni** [comment]
> Hi, @prolajumokeoni Can you please help me? I do not know what I am doing wrong.

- [ ] Kindly add the right content to your [linters.yml ](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) file in your .github/workflows folder

**Comment by Zelalem1222** [comment]
@prolajumokeoni 
Is there any video or something that I can see? I am not understanding anything.

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni Is there any video or something that I can see? I am not understanding anything.

Microverse linters         |  Zelalem linter config  
:-------------------------:|:-------------------------:
Screenshot_30  |  Screenshot_29

Kindly check the previous comment in my [summary](https://github.com/Zelalem1222/Hello-Microverse/pull/1#pullrequestreview-945685700)

### What you need to do ⬇️ 
- Create a .github folder ➡️  Create a workflows folder in your  .github folder ➡️ then create a linters.yml file and add the content  your folder structure should look  like this [.github](https://github.com/microverseinc/linters-config/tree/master/html-css/.github)/[workflows](https://github.com/microverseinc/linters-config/tree/master/html-css/.github/workflows)

**Comment by Zelalem1222** [comment]
@prolajumokeoni Is it fixed now?

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni Is it fixed now?

No, it's not fixed yet

**Comment by Zelalem1222** [comment]
@prolajumokeoni is it working now? and can I create new pull request?

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+h1 {
+  color: cadetblue;
+}
```
Comment (danmainah) [inline styles.css:4]: - [ ] [OPTIONAL] It's always good practice to use classes instead of HTML elements.

**Review by danmainah — APPROVED** [review]
Hi @Zelalem1222 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**